x-pack/platform/plugins/private/transform/public/alerting/transform_health_rule_type/transform_selector_control.tsx

```diff
@@ -50,6 +53,17 @@ export const TransformSelectorControl: FC<TransformSelectorControlProps> = ({
     );
   }) as Exclude<EuiComboBoxProps<string>['onChange'], undefined>;
 
+  const onCreateOption = allowCustomOptions
+    ? (((searchValue, existingOptions) => {
+        if (
+          searchValue.includes('*') &&
+          !existingOptions.some((option) => option.label === searchValue)
```

Comment:
Do we need this additional check? It seems to already be derived from `allowCustomOptions`

Reply:
It's indeed redundant, had added this check before introducing the `allowCustomOptions`. Removed in c250dc49a253a2f95507e7ff59f1d2b2135316a5.